### PR TITLE
Governance: Avoid extra elections if a PMC member steps down just aft…

### PR DESCRIPTION
**Pull request by roidelapluie**
…er the elections

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by nibalizer** [comment]
LGTM

**Comment by roidelapluie** [comment]
Se we have 3 PMC members who did +1, can we merge?

**Comment by daenney** [comment]
I'm not OK with this change. It means that a single person can end up being in charge for an extended period of time.

**Comment by bbriggs** [comment]
I think I have an idea of what you mean, but can you please elaborate?

**Comment by daenney** [comment]
Well, if 4 people step down that means there's one person left. Since there's no boundary at which point we say a re-election is necessary it can mean that 1 person ends up being the PMC for months to the whole cycle.

**Comment by bbriggs** [comment]
I see. Let's say that our minimum number of members is 3 (or maybe set max vacancies at 2). Anything beyond that threshold triggers an automatic election to fill the remaining seats for the term.

**Comment by daenney** [comment]
Ya, that seems reasonable. If you lose the majority of elected members we should revote.

File: share/governance.md [hunk 1]
```diff
@@ -96,7 +96,8 @@ PMC Removal
 
 A member of the PMC can be removed before the end of their 1 year term by a supermajority (2/3) vote of the PMC. This can happen for a number of reasons, but the expected reason is following a Code of Conduct violation incident. Any report of problematic behaviour of PMC members will trigger an investigation and can trigger a vote. A report of the incident and actions taken (if any) will be published for the public. A member of the PMC can step down at any time.
 
-If a vacancy is left on the PMC, the PMC can optionally instruct the elections officer to perform another election or wait until the next election cycle.
+If a vacancy is left on the PMC, the PMC can optionally instruct the elections officer to perform another election, wait until the next election cycle, or use the results of the previous election to select a new member, if the previous election is recent enough.
```
Comment (bbriggs) [inline share/governance.md]: Reword to trigger automatic election to fill remaining seats for the term if ${members.count} < 3

Comment (bbriggs) [inline share/governance.md]: Or as @daenney worded it, "if you lose the majority"

**Comment by hunner** [comment]
Members of the PMC would be elected originally under the mandate of doing what is beneficial for VP. If four people stepped down, it would be responsible and prudent of them to first instruct the elections officer to perform another election as this states. I don't think that we need verbiage covering this explicitly since goodwill is implied throughout.

**Comment by bbriggs** [comment]
I'm going to say we're using the principle of [Hanlon's Razor](https://en.wikipedia.org/wiki/Hanlon%27s_razor) here and not accusing anyone of malice, but just saying, "Maybe it didn't go like we planned"

**Comment by daenney** [comment]
And what if 4 people died in a plane crash? Yes I know unlikely, and yes we're assuming goodwill throughout, but I would place some limitation on what you can reasonably do if you can no longer claim majority representation of the community.

**Comment by hunner** [comment]
Hah, if four of the five members die in a plane crash and the remaining member refuses to hold an election... that would probably violate Hanlon's Razor :D (which is @bbriggs++ for remembering).

**Comment by daenney** [comment]
Sure, but then there's no need to add this language. We're explicitly granting powers to do this here. Instead either limit it, or cut it out which then makes it an issue the community will have to deal with should this ever happen.

I don't think just blanket granting this privilege is a good thing.

**Comment by hunner** [comment]
Guess my nefarious plans are foiled!~

Making daenney's suggested change doesn't weaken the change at all, so +1 then.

**Comment by bbriggs** [comment]
So it looks like we have unanimous approval once @daenney's recommendation is incorporated

**Comment by roidelapluie** [comment]
This is NOT my change guys.

My change is to prevent extra elections if a vacancy is left in the PMC just after an election.

You are speaking about a problem in the ORIGINAL text, where the PMC does not HAVE TO replace the vacancies.

**Comment by roidelapluie** [comment]
My proposal is to make daenney change another PR.

**Comment by daenney** [comment]
Yes this is about your change. You specify that if a vacancy is left after the elections we don't have to re-elect. But like I said, there's no minimum there. So if 4 people bail right after the election, then what? We don't re-elect?

**Comment by roidelapluie** [comment]
@daenney Text from before my change says that we do not need to reelect: -If a vacancy is left on the PMC, the PMC can optionally instruct the elections officer to perform another election or wait until the next election cycle.

My charge means that if you do not reelect you can use the result of last elections. e.g if one PMC member steps down today we could have eputnam to the PMC without new elections.

**Comment by bbriggs** [comment]
But it's still optional. We don't want 4 people to step down and the one person left "opts" to not re-elect.

**Comment by daenney** [comment]
Your example is strange to me. You say that you can reuse someone from the previous elections, but that to me is not the same as getting someone that was on the ticket but not elected to join.

If it's from a previous committee, then sure, maybe.

**Comment by bbriggs** [comment]
So I don't support your change as originally written, no. I really want to see an automatic election triggered _if we dip below a certain threshold of occupied seats_. On a 5 member board, that leaves two seats at the discretion of the board to leave empty.

**Comment by hunner** [comment]
Peeps, @roidelapluie is trying to make an (imho appropriate) amendment and is being requested to incorporate other changes that are not topical to the original intention.

For the original change, @bastelfreak, @Hunner, and @nibalizer approved of it, and @bbriggs has disapproved, and @dhoppe is absent.

The governance document has a long long way to go to being bulletproof; the document doesn't EVEN have a governance change approval process. Lets review changes based on their merit, not on their comprehensiveness.

**Comment by bbriggs** [comment]
Well at the very least we have a majority. Let it sit for a day?

**Comment by bbriggs** [comment]
That should give @dhoppe time to weigh in, even though it won't change the majority vote should he oppose.

**Comment by hunner** [comment]
@bbriggs There isn't even a provision for "unanimous" vs. "majority" at the moment, so we just have to figure this out together for the time being :).

I have high hopes of overhauling the governance doc more in the style of an evolved Nomic ruleset such that it has process for appealing, proposing, ratifying, and amending the bylaws as well as giving them structure and precedence. :D

**Comment by bbriggs** [comment]
No, there isn't, I just want him to be heard

**Comment by dhoppe** [comment]
I agree with @bbriggs and @daenney. We should be able to reach the majority of three people.

**Comment by hunner** [comment]
@dhoppe Welcome to the party!

Okay, as the governance document is not super-solid with respect to ratifying changes, we should probably figure that out first before adding changes for handling people stepping down (which hasn't happened yet and hopefully won't).

 http://legacy.earlham.edu/~peters/writing/nomic.htm#initial%20set rules 104 and 203 may be of interest :).

**Comment by bbriggs** [comment]
TIL that Hunner's whole plan was to create a really elaborate and high-stakes Nomic :)

**Pull request closed without merging**